torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -561,6 +571,7 @@ static void PrepareForRemoveMutations(MutationRemover& mr, Block* b) {
       if (!mr.inplaceOpVariant(node)) {
         continue;
       }
+      ChangeInputTypeForInplaceOps(node);
```

Comment:
Rename the function and mention implicit cast

Reply:
Changed function name and add an example